Source/Core/Core/State.cpp

```diff
@@ -525,13 +493,69 @@ void SaveAs(const std::string& filename, bool wait)
       true);
 }
 
+static bool GetVersionFromLZO(StateHeader& header, File::IOFile& f)
+{
+  // Just read the first block, since it will contain the full revision string
+  lzo_uint32 cur_len = 0;  // size of compressed bytes
+  lzo_uint new_len = 0;    // size of uncompressed bytes
+  std::vector<u8> buffer;
+  buffer.resize(header.must_be_zero);
+
+  if (!f.ReadArray(&cur_len, 1) || !f.ReadBytes(out, cur_len))
+    return false;
+
+  const int res = lzo1x_decompress(out, cur_len, buffer.data(), &new_len, nullptr);
+  if (res != LZO_E_OK)
+  {
+    // This doesn't seem to happen anymore.
+    PanicAlertFmtT("Internal LZO Error - decompression failed ({0}) ({2}) \n"
+                   "Unable to retrieve outdated savestate version info.",
+                   res, new_len);
```

Comment:
This fmt string is not legal, I'm surprised this wasn't caught by a compiletime assert somewhere. There is no third argument matching `{2}`.

Reply:
I can't believe I missed that, good catch! This was left in after I modified the old LZO panic alert.